packages/ag-charts-community/src/util/document.ts

```diff
@@ -0,0 +1,7 @@
+export function onContentLoaded(handler: CallableFunction) {
+    if (typeof document === 'undefined' || document.readyState === 'complete') {
+        handler();
+    } else {
+        window.addEventListener('DOMContentLoaded', () => handler());
```

Comment:
Please avoid use of the global `window` - it doesn't play well with server-side rendering.

We were passing the `window` and `document` to `AnimationManager` previously for this use-case, could we continue with that approach for consistency please?